packages/launcher/lib/browsers.ts

```diff
@@ -9,6 +9,7 @@ export const debug = Debug('cypress:launcher:browsers')
 /** starts a found browser and opens URL if given one */
 export type LaunchedBrowser = cp.ChildProcessByStdio<null, Readable, Readable>
 
+// NOTE: firefox uses geckodriver to launch the browser
```

Comment:
Just to clarify Firefox doesn't use geckodriver but it's the other way around. So maybe the following change would be clearer:

```suggestion
// NOTE: For Firefox geckodriver is used to launch the browser
```

Reply:
updated in fdcde2dd7c